packages/sku/sku-types.d.ts

```diff
@@ -39,16 +39,16 @@ interface SkuRouteObject {
   route: string;
   name?: string;
   entry?: string;
-  languages?: string[];
+  languages?: ReadonlyArray<string>;
```

Comment:
If you prefer the other syntax:

```suggestion
  languages?: readonly string[];
```

Reply:
Was aiming for consistency with the others here so I know where all my readonlys are